### ISSUE TITLE
ReactiveSwift & ReactiveCocoa fails to generate documentation

### ISSUE THREAD

**Issue by Burgestrand** [CocoaPods/cocoadocs.org#506]
ReactiveSwift v1: http://cocoadocs.org/docsets/ReactiveSwift/1.0.0/
ReactiveCocoa v5: http://cocoadocs.org/docsets/ReactiveCocoa/5.0.0/

ReactiveSwift appears to have their documentation generated by Jazzy, as can be seen on the documentation site here: http://reactivecocoa.io/reactiveswift/docs/latest/

**Comment by Burgestrand** [comment]
Asking in ReactiveSwift's IRC, somebody dug up the following error from http://api.cocoadocs.org:4567/error/ReactiveSwift/1.0.0:

```
report_error({"message":"Unable to find an Xcode with swift version 3.0.2.","trace":["/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/sourcekitten.rb:140:in `run_sourcekitten'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/podspec_documenter.rb:31:in `block (3 levels) in sourcekitten_output'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/podspec_documenter.rb:26:in `map'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/podspec_documenter.rb:26:in `block (2 levels) in sourcekitten_output'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/podspec_documenter.rb:21:in `chdir'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/podspec_documenter.rb:21:in `block in sourcekitten_output'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/gems/cocoapods-1.1.1/lib/cocoapods/config.rb:41:in `with_changes'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/podspec_documenter.rb:17:in `sourcekitten_output'","/Users/cocoadocs/.rvm/gems/ruby-2.1.3/bundler/gems/Jazzy-7e752d129a76/lib/jazzy/doc_builder.rb:61:in `build'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:431:in `document_spec'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:520:in `document_spec_at_path'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:152:in `block in doc'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:147:in `each'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:147:in `doc'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:219:in `block in cocoadocs_url'","/Users/cocoadocs/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/open-uri.rb:153:in `open_uri'","/Users/cocoadocs/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/open-uri.rb:704:in `open'","/Users/cocoadocs/.rvm/rubies/ruby-2.1.3/lib/ruby/2.1.0/open-uri.rb:34:in `open'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:216:in `cocoadocs_url'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:167:in `cocoadocs'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:102:in `initialize'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:552:in `new'","/Users/cocoadocs/cocoadocs.org/./cocoadocs.rb:552:in `<main>'"]}
)
```

**Comment by andersio** [comment]
It _seems_ CocoaDocs does not work when `.swift-version` specifies the minor version. 

Related: https://github.com/ReactiveCocoa/ReactiveSwift/pull/266

**Comment by orta** [comment]
Aye, I probably need to install a new Xcode on the cocoadocs server. I'm probably not going to have time for at least a month for this as I'll be on a honeymoon. An easy fix ATM is to make CocoaDocs change "3.0.2" -> "3.0.1" before passing it to jazzy

**Comment by orta** [comment]
Fixed by #508 - 

- http://cocoadocs.org/docsets/ReactiveSwift/1.1.3/
- http://cocoadocs.org/docsets/ReactiveCocoa/5.0.3/
